mkdocs.yml

```
@@ -527,6 +527,7 @@ nav:
         - Ampli Codegen: data/sdks/typescript-react-native/ampli.md
       - Flutter:
         - data/sdks/flutter/index.md
+        - data/sdks/flutter/migration.md
```

```suggestion
       # - data/sdks/flutter/migration.md
```

Would like to hold off on listing this until the new version 4.0 doc is ready.

---

Makes sense! Even though it's not visible on nav bar, customers can still access it by url. Update it at https://github.com/amplitude/amplitude-dev-center/pull/239/commits/57333993c74bc034be1119c45fb4673deaaae36b